src/containers/Profile.jsx

```diff
@@ -37,7 +37,11 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
     dispatch(updateInfo(field, value))
   },
   requestExport: async () => {
-    await dispatch(requestExport())
+    try {
+      await dispatch(requestExport())
```

Comment:
You should move the line `Alerter.success(ownProps.t('ProfileView.export.success'))` here, to avoid showing the success alerter when there is an error

Reply:
Oh, forgot it. Thanks 👍